terraform/deploy/terraform.tfvars.j2

```diff
@@ -35,8 +35,3 @@ github_vpc = {
 {%- for k, v in ucfs['github']['vpc'].items() %}
   {{ k }} = "{{ v }}"{% endfor %}
 }
-
-packer_egress_test_zip = {
-  base_path = "../../../packer-egress-test-release"
-  version   = "0.0.12"
-}
```

Comment:
don't we still need this for running tests in the pipeline?

Reply:
It requires the zip file to be present.  The GHA is ephemeral and does not have the zip file.  
It's only used when creating AWS-Concourse, to run a test in Concourse.  We're no longer using Concourse